src/components/utils/caseUtils.ts

```diff
@@ -65,12 +74,16 @@ export function useDownloadUtils() {
                 .get('Content-Disposition')
                 .split('filename=')[1];
             filename = filename.substring(1, filename.length - 1); // We remove quotes
+            const fileExtension = filename.split('.').pop();
```

Comment:
Add an enpoint like` network-converion-server` with filename (without extension)
and get extension from the back

Reply:
we do not have extension, we have only format which is not the same: 
Example 
 format : MATPOWER
 extension : mat
I think that the only solution to get format is to extract it from original case name